ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -1897,6 +1936,23 @@ private void setUpKsqlResource() {
     );
   }
 
+  private ServiceContext createTestServiceContext() {
+    final ServiceContext sc = spy(TestServiceContext.create(kafkaTopicClient));
+    serviceContexts.add(sc);
+
+    doAnswer(inv -> {
+      Object m = inv.getMock();
+      if (serviceContexts.contains(m)) {
+        serviceContexts.remove(m);
```

Comment:
what are we using this for? I don't see any tests that require us to properly maintain the list of service contexts?

Reply:
It's not testing anything. I used just to remove objects from the list of serviceContexts. Do you think this is not necessary? I suspect now that it might not, as this is just a test class, and list is cleared on every new test case.

Reply:
I will remove this code instead of fixing it. Thes are just tests. The serviceContexts list is cleared on every test, so it is harmless to leave them there.